src/tests/EdFi.DataManagementService.Tests.E2E/Features/RESTAPIConventions/POSTRequest.feature

```diff
@@ -0,0 +1,258 @@
+Feature: Validating POST Actions
+
+        @APIConventions @POST
+        Scenario: 01 Verify user can send a POST using a valid data
```

Comment:
Some of these tests could be similar to the existing ones in other feature files, but in order to keep the order of the sections we are covering it would be a good idea to keep them.

Reply:
This seems like a lot of duplication of the features in `CreateResourcesValidation.feature`. If there are any scenarios here that we do not already cover, then please add them to `CreateResourcesValidation.feature` instead of having a new file.

The goal of these tickets is to make sure all of the functionality described in the API standards guidance has scenarios. If it is convenient to organize the scenarios to more closely match the structure of the API standards, then let's think about re-organizing the existing files instead of creating duplicate tests.

Remember: every test that runs take a little bit of time and means more waiting to see if all tests passed.

Reply:
Make sense, the new scenarios were added in the `CreateResourcesValidation.feature `, also the scenarios were tagged with the @POST value.